docs/src/model_docs/lateral/local-inertial.md

```diff
@@ -46,8 +46,16 @@ inertial_flow_alpha = 0.5         # alpha coefficient for model stability (defau
 froude_limit = true               # default is true, limit flow to subcritical-critical according to Froude number
 h_thresh = 0.1                    # water depth [m] threshold for calculating flow between cells (default = 1e-03)
 riverlength_bc = 1000.0           # river length [m] for boundary points (default = 1e04)
+riverdepth_bc = 1.5               # river depth [m] for boundary points (default = 0.0)
 floodplain_1d = true              # include 1D floodplain schematization (default = false)
 ```
+It is also possible to provide the `riverlength_bc` and `riverdepth_bc` parameters through
+the model parameter NetCDF file, as follows:
+```toml
+[input.lateral.river]
+riverlength_bc = "riverlength_bc"
+riverdepth_bc = "riverdepth_bc"
```

Comment:
Yes, I agree this approach can be a bit confusing. It was mainly done to avoid a breaking release. I think we should go only for the netCDF approach then (also related to your comments below about the handling of these parameters in the code). Do you agree?

Reply:
I agree with moving these options to be only in the `[input.lateral.river]` section!

Reply:
See commits https://github.com/Deltares/Wflow.jl/pull/371/commits/ed9233ffcd46592482c3f8400b577ab2ad7fffc2 and https://github.com/Deltares/Wflow.jl/pull/371/commits/3bacd13923e0ea71942098aa6ff5bf5aa032f896.